test/known_problems/should_pass/update_map_type_error.erl

```diff
@@ -4,12 +4,6 @@
 
 -spec t(list()) -> map().
 t(Rows) ->
-    lists:foldl(fun({Id, Name, Member}, #{members := []} = List) ->
-                        List#{id := Id,
-                              name := Name,
-                              members := [binary_to_list(Member)]};
-                   ({_, _, Member}, #{members := Members} = List) ->
-                        List#{members := [binary_to_list(Member) | Members]}
-                end, #{id => undefined,
-                       name => undefined,
-                       members => []}, Rows).
+    lists:foldl(fun(Id, List) ->
+                        List#{id := Id}
```

Comment:
This could have a less confusing name, e.g. `Map`.

Reply:
Errr.. Indeed :D